src/sentry/utils/data_scrubber.py

```diff
@@ -46,17 +46,24 @@ class SensitiveDataFilter(object):
     Asterisk out things that look like passwords, credit card numbers,
     and API keys in frames, http, and basic extra data.
     """
-    # http://www.richardsramblings.com/regex/credit-card-numbers/
-    VALUES_RE = re.compile(r'\b(?:3[47]\d|(?:4\d|5[1-5]|65)\d{2}|6011)\d{12}\b')
+    VALUES_RE = re.compile(r'|'.join([
+        # http://www.richardsramblings.com/regex/credit-card-numbers/
+        r'\b(?:3[47]\d|(?:4\d|5[1-5]|65)\d{2}|6011)\d{12}\b',
+        # various private/public keys
+        r'-----BEGIN[A-Z ]+(PRIVATE|PUBLIC) KEY-----.+-----END[A-Z ]+(PRIVATE|PUBLIC) KEY-----',
+        # social security numbers (US)
+        r'^\b\d{3}-?\d{2}-?\d{4}\b',
```

Comment:
This is going to be too overly eager imo, since it'll reduce to blocking any 9 digit number.

We could refine this like our credit card regex by only checking actual valid SSNs: https://en.wikipedia.org/wiki/Social_Security_number#Valid_SSNs

I saw `^(?!(000|666|9))\d{3}-(?!00)\d{2}-(?!0000)\d{4}$` on the internet, but this would need to be checked a bit closer to make sure. Granted, this doesn't look like it'd filter down much, unless we made hyphens required.